debian/cloud-init-base.config

```diff
@@ -51,13 +51,21 @@ get_yaml_list() {
 migrate_debconf_to_cloud_init_base() {
     local old="cloud-init/$1" new="cloud-init-base/$1"
     if db_get "$new"; then
-       # cloud-init-base/* win against cloud-init/* values
-       return 0
+        if db_get "$old"; then
+            # Unset cloud-init/* because cloud-init-base can't unregister
+            # cloud-init debconf values. db_set exits 0 but makes no debconf
+            # changes.
+            db_set "$old" "" || :
+        fi
+        return 0
```

Comment:
this could just be `return`, right? same comment as below

Reply:
yes this could. I instead reduced the conditional statement to be if db_get... elif to avoid performing the early return anyway.